src/main/java/com/amazon/opendistroforelasticsearch/security/configuration/PrivilegesInterceptorImpl.java

```diff
@@ -170,18 +179,30 @@ public Boolean replaceKibanaIndex(final ActionRequest request, final String acti
 
         }
 
+        return CONTINUE_EVALUATION_REPLACE_RESULT;
+    }
+
+    private CreateIndexRequest checkAndCreateRequestIfAbsent(final String name) {
```

Comment:
Can we rename this to just "createIndexRequestIfAbsent" ? not clear from the name what the check is